beesdoo_shift_welcome_screen/views/welcome.xml

```diff
@@ -35,8 +35,8 @@
             <div class="row align-items-center">
                 <div class="col-md-3">
                     <img
-                        t-if="partner.image"
-                        t-attf-src="data:image/*;base64,{{partner.image}}"
+                        t-if="partner.image_1920"
+                        t-attf-src="data:image/*;base64,{{partner.image_1920}}"
```

Comment:
i guess this is not a full-screen image, so image_1024 or image_512 should be enough (and faster to load).